aerich/__init__.py

```diff
@@ -204,7 +212,7 @@ async def downgrade(self, version: int, delete: bool, fake: bool = False) -> lis
         if version == -1:
             versions = [specified_version]
         else:
-            versions = await Aerich.filter(app=self.app, pk__gte=specified_version.pk)
```

Comment:
I also changed this line from gte to gt, as it seemed strange to me that "downgrading to certain version" also ran downgrade for chosen version, so you actually got base at "chosen migration - 1"

Although if you have counterpoints to why it should stay the way it were, I can rollback it

Reply:
I agree that the downgrade command seems strange.
Could you roll back the unnecessary changes? Let’s keep things simple, zero in on the new feature. We can move the refactoring work to another pull request (PR).